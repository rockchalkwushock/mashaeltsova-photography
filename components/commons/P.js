import PropTypes from 'prop-types'

const P = ({ children, className }) => (
  <p className={className}>
    {children}
    <style jsx>{`
      p {
        margin: 0;
        padding: 0;
      }
      .services {
        -webkit-box-flex: 1;
        -ms-flex: 1 1 auto;
        flex: 1 1 auto;
      }
    `}</style>
  </p>
)

P.propTypes = {
  className: PropTypes.string
}

export default P
