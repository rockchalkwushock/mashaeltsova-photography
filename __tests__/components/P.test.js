import React from 'react'
import { render } from 'enzyme'

import P from '../../components/commons/P'

describe('Component: P', () => {
  test('Renders without exploding as .services', () => {
    const tree = render(<P className="services" />)
    expect(tree).toMatchSnapshot()
  })
  test('Renders without exploding', () => {
    const tree = render(<P />)
    expect(tree).toMatchSnapshot()
  })
})
