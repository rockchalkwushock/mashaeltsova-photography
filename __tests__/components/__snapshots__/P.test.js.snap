// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: P Renders without exploding 1`] = `
<p
  class="jsx-3204703105 "
/>
`;

exports[`Component: P Renders without exploding as .services 1`] = `
<p
  class="jsx-3204703105 services"
/>
`;
